tmpl/wiki/weekly/2015-01-28.md

```diff
@@ -0,0 +1,79 @@
+28th Jan 2014: OPAM 1.2, FOSDEM, new URL and 3.0 meta-planning
+
+### Agenda ###
+
+- Move to only OPAM 1.2
+- FOSDEM demo
+- TLS merge progress
+- Turning on mirage.io?
+- Planning for 3.0
+
+Attendees: 
+Amir Chaudhry, David Kaloper, Thomas Leonard, Jon Ludlam, Hannes Mehnert,
+Richard Mortier, Dave Scott and Mindy Preston
+
+
+### Notes ###
+
+#### OPAM 1.2 support only ####
+
+ThomasG submitted a patch for the `mirage` tool to enable the switch to
+OPAM 1.2 only. The patch should report an error and ask users to upgrade
+OPAM 1.2. This matters as the upstream distros are still on OPAM 1.1 (e.g
+Ubuntu).  we still need to decide what we want and it would be good to make
+the patch user controlled (e.g user can override it if they want).
+
+#### FOSDEM demo ####
+
+Amir is heading to FOSDEM this coming weekend and has planned a demo around
+serving a game from the cubieboard (via setting up a wifi access point).
+Having issues with the wifi bridge but not sure what the problem really is.
+Will need to get this working if there's to be a demo.
+
+#### TLS Merge progress ####
+
+This is waiting on DavidK, who has to review and merge TLS patches. Recently
+been playing with a TLS unikernel on Xen and working towards a paper. Will
+also be inviting people to break it soon.
+
+Still have the entropy story to communicate and are starting with xentropyd.
+Should be done in a few days with patch reviews.
+
+ThomasL has a few other changes to the FLOW interface that need to be dealt
+with (Error messages in a few places). There's an issue on mirage-dev before
+there can be releases. There might also be more changes to libs/interfaces
```

Comment:
This issue is tracking the status: https://github.com/mirage/mirage-dev/pull/52
Most components can be released with the updates now, but some need to be done together (e.g. TLS both provides and uses the new interface). It would be possible to make all the updates decoupled if preferred (e.g. first updating tls to provide the new interface, then releasing the new interface, then updating tls a second time to use the new API).